src/utilities.py

```diff
@@ -226,10 +226,9 @@ def check_if_floats(array):
     return True
 
 
-def get_config_path() -> str:
-    if os.getenv("XDG_CONFIG_HOME"):
-        return os.path.join(os.getenv("XDG_CONFIG_HOME"), "Graphs")
-    return os.path.join(str(Path.home()), ".local", "share", "Graphs")
+def get_config_directory():
```

Comment:
Interesting with this implementation from GFile. Does this work the same for Flathub environments and for source builds?

While compiling from source is not really supported, it would be nice if it works either way.

Reply:
`GLib.get_user_config_dir()` does pretty much the same as `get_config_path()` used to do (excluding the subdir "Graphs") with the added benefit of working on macOS and windows as far as I can tell. This function simply creates a `GFile` for that path and returns the subdir "Graphs" of that file.